fast64_internal/sm64/sm64_objects.py

```diff
@@ -2156,8 +2156,21 @@ def draw_export_options(self, layout):
             col.prop(self, "export_bhv")
             self.draw_obj_name(layout)
 
+    @property
+    def actor_names(self) -> list:
+        return list(dict.fromkeys(filter(None, [self.obj_name_col, self.obj_name_gfx])).keys())
+
+    @property
+    def export_locations(self) -> str | None:
+        names = self.actor_names
+        if len(names) > 1:
+            return f"({'/'.join(names)})"
```

Comment:
Note `{a,b,c}` is a more common syntax for indicating "a or b or c", as it's shell syntax. For example
```c
$ echo foo/{a,b,c}
foo/a foo/b foo/c
```